CHANGELOG.rst

```diff
@@ -3,6 +3,12 @@ Changelog for Mantarray Desktop App
 0.4.3 (2021-03-30)
```

Comment:
can you bump this version to match the software version

Reply:
well that's embarrassing, fixed it now